p2p/constants.py

```diff
@@ -62,3 +62,6 @@
 
 # Maximum peers number, we'll try to keep open connections up to this number of peers
 DEFAULT_MAX_PEERS = 25
+
+# Maximum allowed depth for chain reorgs.
+MAX_REORG_DEPTH = 24
```

Comment:
Is 24 used by other clients? If I was making up a number, it would be bigger. But also it would be nice to have a framework to decide a reasonable reorg maximum.

Reply:
No, geth uses a somewhat complicated algorithm that I didn't understand well. I filed #978 so that we don't forget about this